Vagrantfile

```diff
@@ -0,0 +1,28 @@
+Vagrant.configure("2") do |config|
+  # The box / OS image
+  config.vm.box = "ubuntu/trusty64"
+  # Network
+  config.vm.network :forwarded_port, guest: 8000, host: 8080
+  config.vm.network :private_network, ip: "192.168.33.10"
+
+  # setup
+  config.vm.provision "shell", inline: <<-SHELL
+     apt-get update
+     sudo apt-get install -y sqlite3 libsqlite3-dev python3-pip git
+     pip3 install -U pip setuptools wheel
+     git clone https://github.com/openwisp/openwisp-network-topology/
+     echo "installing openwisp-network-topology"
+     cd openwisp-network-topology
+     pip install -U .

```

Comment:
I had problem because its installing django2, when tests compitable only with django1 (**default: ImportError: No module named 'django.core.urlresolvers'**)
So I added line `pip3 install django~=1.0` and it helped me